frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import * as History from 'history';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Popover, PopoverPosition } from '@patternfly/react-core';
+import { Button } from 'patternfly-react';
+import { ErrorCircleOIcon, InProgressIcon, OkIcon, InfoAltIcon } from '@patternfly/react-icons';
+import {
+  global_danger_color_100 as dangerColor,
+  global_success_color_200 as successColor,
+} from '@patternfly/react-tokens';
+
+import { CamelCaseWrap } from '@console/internal/components/utils';
+import { DASH } from '../../constants';
+
+type StatusProps = {
+  icon?: React.ReactElement;
+  title?: string;
+  spin?: boolean;
+};
+
+export const StatusIconAndText: React.FC<StatusProps> = ({ icon, title, spin }) => {
+  if (!title) {
```

Comment:
If there is an icon but no title, you don't want to show the icon?

Reply:
I don't think we want to show icon when there is no title. I am sticking with the original StatusIconAndText logic: https://github.com/openshift/console/blob/master/frontend/public/components/utils/status-icon.tsx#L72-L74

Reply:
ok